rust/guest_wrapper/src/lib.rs

```diff
@@ -8,19 +8,19 @@ mod private {
 
 include!(concat!(env!("OUT_DIR"), "/guest_id.rs"));
 
-#[cfg(not(clippy))]
```

Comment:
This should stay. I only meant swapping `#[cfg(not(clippy))]` for `#[allow(unused)]` for the const fn.